modules/videoio/cmake/detect_ximea.cmake

```diff
@@ -1,6 +1,6 @@
 if(NOT HAVE_XIMEA)
   if(WIN32)
-    get_filename_component(regpath "[HKEY_CURRENT_USER\\Software\\XIMEA\\CamSupport\\API;Path]" ABSOLUTE)
+    get_filename_component(regpath "[HKEY_LOCAL_MACHINE\\SOFTWARE\\XIMEA\\API_SoftwarePackage;Path]" ABSOLUTE)
```

Comment:
This change breaks behavior of this script.

Proper flow should be:
- try `HKEY_CURRENT_USER` key
- if empty, then try `HKEY_LOCAL_MACHINE`